utils/responseUtil.ts

```diff
@@ -26,6 +26,15 @@ export class ResponseUtil {
         res.status(400).json({ error: message });
     }
 
+    /**
+     * Method for returning a status 401 Bad Request response
```

Comment:
typo in comment - should be `401 Unauthorized`

Reply:
Good catch!